### PR TITLE
[dcl.init] It is the 'initializer' who specifies an initial value for…

### PR DESCRIPTION
**Pull request by Heuristack**
… its 'preceding declarator' (#1615)

### CODE REVIEW

File: source/declarators.tex [hunk 1]
```diff
@@ -2388,7 +2388,7 @@
 \indextext{initialization|(}
 
 \pnum
-A declarator can specify an initial value for the
+An initializer can specify an initial value for the
```
Comment (tkoeppe) [inline source/declarators.tex]: Why "can"? Is it possible for an initializer to *not* specify an initial value?

**Comment by jensmaurer** [comment]
Note the comments from @zygoloid in my competing pull request #1616. We certainly should not merge the present pull request as-is.

File: source/declarators.tex (hunk 1 above)
Comment (Heuristack) [inline source/declarators.tex]: My understanding here is "can" means "can be used", but obviously it is not clear enough.

**Comment by Heuristack** [comment]
Hi @jensmaurer, my proposal is from the perspective of respecting the original text and author, and minimize the change and maximize consistency. However, since you have done a larger rewrite, I will close it when your change is merged.

But without considering a rewrite, I believe the my proposed change is a proper one. Thanks!

**Comment by zygoloid** [comment]
#1616 merged, which also addresses this.

**Pull request closed without merging**